docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,372 @@
+You can now copy over `~/copytob.tar.gz` to a writeable volume.
+
+For ease of use, let us make a folder called `bconfig` and copy it over.
+
+(You might need to use an admin or sudo permissions)
+
+eg (These will be different for LiveUSB's)
+`sudo mkdir /media/{yourusername}/writable/bconfig`
+`sudo cp ~/copytob.tar.gz /media/{yourusername}/writable/bconfig`
+
+Now unmount the disk from your system.
+**This will be the last time you will ever connect this disk to a system with internet access.**
+
+Now boot `Wallet-B` with the disk inserted **(DO NOT CONNECT YOUR ETHERNET CABLE)**.
+Depending on the OS it should show you a system setup and a default user creation menu, proceed with a strong password.
+Then once the system is installed it should reboot.
+
+Once logged in get familiar with the system, adjust the clock, etc, and then open a terminal.
+
+??? note "Raspberry Pi specific (click to expand)"
+
+    Optionally you may choose to disable WiFi and Bluetooth at the kernel level too. 
+
+    Create a file in
+
+    ```no-highlight
+    /etc/modprobe.d/raspi-blacklist.conf
+    ```
+
+    and add the contents
+
+    ```no-highlight
+    blacklist brcmfmac
+    blacklist brcmutil
+    blacklist hci_uart
+    blacklist btbcm
+    blacklist btintel
+    blacklist rfcom
+    blacklist btqca
+    blacklist btsdio
+    blacklist bluetooth
+    ```
+
+Disable WiFi and Bluetooth using rfkill:
+
+```no-highlight
+sudo rfkill block wifi
+sudo rfkill block bluetooth
+```
+
+Now to setup the ufw rules run the following commands.
+
+```no-highlight
+sudo ufw default deny outgoing
+sudo ufw default deny incoming
+sudo ufw allow out to {LOCAL_IP_OF_COMPUTER-A} port 9109
+sudo ufw enable
+```
+
+Replace {LOCAL_IP_OF_COMPUTER-A} with the IP address of `Computer-A`
+
+Now you may connect the ethernet cable.
+Wait for the connection to take place and then test if you can connect to `Computer-A` on port 9109
+
+wget {LOCAL_IP_OF_COMPUTER-A}:9109
+
+You should get an error a 400 bad request error. This is fine and shows that a connection is possible.
+
+Now to setup dcrwallet.
+
+First, let us check that the tarfile is unchanged.
+
+`sha256sum /bconfig/copytob.tar.gz`
+
+Once the hash is matched then you can extract it.
+
+```no-highlight
+mkdir ~/decredconfigs
+mkdir ~/decred
+tar -xf /bconfig/copytob.tar.gz -C ~/decredconfigs
+tar -xf ~/decredconfigs/decred-linux-{amd/arm}64-v{{ cliversion }}.tar.gz -C ~/decred
+cp ~/decred/decred-linux-{amd/arm}64-v{{ cliversion }}/* ~/decred/
+rm -rf ~/decred/decred-linux-{amd/arm}64-v{{ cliversion }}/
+```
+
+Now we have the Decred binaries in the ~/decred/decred-linux-{amd/arm}64-v{{ cliversion }} folder.
+
+Let us setup the dcrctl and dcrwallet config files that will allow it to connect to `Computer-A`.
+
+```no-highlight
+mkdir ~/.dcrwallet/
+mkdir ~/.dcrctl/
+```
+
+Now, these files need to be edited.
+
+First copy the RPC username and password from ~/decredconfigs/dcrd.conf
+
+Let us name them {username} and {password}.
```

Comment:
This line should be changed.
I also think the username and password bit might deserve a tad bit more explanation, but I don't quite know the severity of messing up this step. Is the reader supposed to create their own username and password for these files, or modify them from the `~/decredconfigs/dcrd.conf`, or just copy them directly?

All I know is that I've set up several nodes and wallets on multiple isolated systems connected like this before, and this part is always confusing to me because I'm always concerned about which username and password to use, and if it's important that they be kept ultra-secret on the "vulnerable" node or not.

Reply:
I agree that this can be a bit confusing but I think its a bit out of scope for this document. Would rather link to another page/external resource for more info.